### DISCUSSION TITLE
Project outline following CMAKE_FOLDER

### DISCUSSION THREAD

**Discussion by NekkoDroid** [microsoft/vscode-cmake-tools#2487] [Ideas]
As far as I have been able to figure out there is currently no way for the project outline to follow the `(CMAKE_)FOLDER` of targets (like the CMake target view in Visual Studio). Currently the outline shows the targets relative to the actual folder structure.

Is there any possibility that this could be added?

**Comment by elahehrashedi** [comment]
Thank you for suggesting this feature, it looks like we have a feature request reported for this #491. Please go and click the 👍  to vote for it.
